ensek/client.py

```diff
@@ -44,7 +44,8 @@ def __init__(self, *, api_url, api_key):
         self._headers = {'Authorization': f'Bearer {self._api_key}'}
 
     def create_meter_reading(
-        self, *, account_id, meter_point_id, register_id, value, timestamp
+        self, *, account_id, meter_point_id, register_id, value, timestamp,
+        source=None,
```

Comment:
Why default to none? What happens when source is none? (not in the tests).

Reply:
it wont be included in the request, this is what we want